db/compaction/compaction_job.cc

```diff
@@ -834,13 +833,15 @@ Status CompactionJob::VerifyOutputFiles() {
       // verification as user reads since the goal is to cache it here for
       // further user reads
       ReadOptions verify_table_read_options(Env::IOActivity::kCompaction);
```

Comment:
As was mentioned in a live discussion, we should be respecting configured compaction readahead sizes here. I don't see that.